local/local_test.go

```diff
@@ -133,11 +130,13 @@ func testLocalImage(t *testing.T, when spec.G, it spec.S) {
 			var repoName = newTestImageName()
 
 			it.Before(func() {
-				h.CreateImageOnLocal(t, dockerClient, repoName, fmt.Sprintf(`
-					FROM scratch
-					LABEL repo_name_for_randomisation=%s
-					LABEL mykey=myvalue other=data
-				`, repoName), nil)
+				existingImage, err := local.NewImage(repoName, dockerClient)
```

Comment:
Using `existingImage` instead of `baseImage` (in earlier commit) to match the context and clarify that this is about images that already exist